client.go

```diff
@@ -71,6 +87,10 @@ type Config struct {
 	// needs to be goroutine safe.
 	OnSendMessageBatch func(*sqs.SendMessageBatchOutput, error)
 
+	// DeleteBatchEnabled specifies that delete message dispatcher will
+	// be enabled or not. If not specified, defaults to false.
+	DeleteBatchEnabled bool
```

Comment:
Aha! I believe you're trying to use this package as a drop-in replacement for a standard AWS SDK. `asyncsqs` is meant to be used in conjunction with AWS SDK.

Reply:
Not exectly drop in replacement but helpful if user is using it then don't have to maintain two different places.